packages/kit/src/exports/vite/utils.js

```diff
@@ -206,9 +206,9 @@ export function error_for_missing_config(feature_name, path, value) {
 		return acc.replace(hole, `${indent}${part}: ${rhs}`);
 	}, hole);
 
-	throw new Error(
+	throw stackless(
 		dedent`\
-			To enable \`${feature_name}\`, add the following to your \`svelte.config.js\`:
+			To enable ${feature_name}, add the following to your \`svelte.config.js\`:
 
 			\`\`\`js
```

Comment:
Wondering if we should remove the language tag here since it doesn't do anything.
```suggestion
			\`\`\`
```

Reply:
maybe we should remove the fence altogether, i'm not sure it adds much. any objections @elliott-with-the-longest-name-on-github?